.github/workflows/simd-tier-correctness.yml

```
@@ -1,122 +1,122 @@
-          if (-not $level)
-          {
-              throw "No 'Scanner SIMD level' banner under -${{ matrix.chip }}; the selected tier is unverified"
-          }
-          $observed = $level.Matches[0].Groups[1].Value
-          Write-Host "Observed tier under -${{ matrix.chip }}: $observed"
-          if ($observed -ne "${{ matrix.tier }}")
-          {
-              throw "Expected ${{ matrix.tier }} under -${{ matrix.chip }} but the scanner selected $observed"
-          }
-          if ($rc -ne 0) { throw "ScannerTest failed under -${{ matrix.chip }} (${{ matrix.tier }})" }
+name: Scanner SIMD tier correctness
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)